src/components/Grid/Cell.tsx

```diff
@@ -255,22 +253,6 @@ export default class Cell extends React.Component<Props> {
           style={style}
           onClick={this.handleClick}
           onContextMenu={this.handleRightClick}
-          onTouchStart={(e) => {
-            const touch = e.touches[e.touches.length - 1];
-            this.touchStart = {pageX: touch.pageX, pageY: touch.pageY};
-          }}
-          onTouchEnd={(e) => {
-            if (e.changedTouches.length !== 1 || e.touches.length !== 0) return;
-            const touch = e.changedTouches[0];
-            if (
-              !this.touchStart ||
-              (Math.abs(touch.pageX - this.touchStart.pageX) < 5 &&
-                Math.abs(touch.pageY - this.touchStart.pageY) < 5)
-            ) {
-              e.preventDefault();
-              onClick(this.props.r, this.props.c);
```

Comment:
It looks like this was trying to detect clicks without the 300ms click delay, but this can be done with the touch-action: manipulation property which I've added to the css, which results in immediate click events after touchdown and touchup.